examples/huggingface/huggingface-large-model-inference.ipynb

```diff
@@ -42,7 +42,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install sagemaker --upgrade"
+    "!pip install sagemaker==2.161.0"
```

Comment:
why fixing version will help? Won't --upgrade just upgrade?

Can you change to

```
%pip install sagemaker --upgrade
```

use notebook kernel magic

Reply:
Some old notebook instance, even doing it with --upgrade will not work. So this is to make sure the user has the latest 2.161.0 version. Later I can change it to --upgrade.